### ISSUE TITLE
Best way to implement a high-performing "collapsible section list" - OR Is there a way to avoid rendering content when the accordion is collapsed?

### ISSUE THREAD

**Issue by ryanphung** [oblador/react-native-collapsible#333]
I'm trying to create a "collapsible section list", using `Accordion` and `FlatList`. Basically an `Accordion` where each section of data is rendered as a `FlatList`:

```
Accordion
- FlatList
- FlatList
- FlatList
```

However when there are hundreds of items, I start to notice performance degradation

My attempted solution is to avoid rendering the section that are collapsed. Using the `isActive` flag inside `renderContent`:

```
renderContent = (content, index, isActive) => {
    return (
      isActive ? <FlatList ... /> : null
    )
```

However I notice that the content is hidden *before* the animation starts. Is there any way to hide the content *after the animation ends*, and conversely, render the content *before the animation starts*?

Thank you.

**Comment by ryanphung** [comment]
I searched around more and looks like Pull Request #318 is a good solution to re-solve this

**Comment by oblador** [comment]
Released in 1.6.0
